flink-libraries/flink-sql-client/src/main/java/org/apache/flink/table/client/config/Environment.java

```diff
@@ -102,10 +112,10 @@ public static Environment parse(String content) throws IOException {
 	public static Environment merge(Environment env1, Environment env2) {
 		final Environment mergedEnv = new Environment();
 
-		// merge sources
-		final Map<String, Source> sources = new HashMap<>(env1.getSources());
-		mergedEnv.getSources().putAll(env2.getSources());
-		mergedEnv.sources = sources;
+		// merge tables
+		final Map<String, TableDescriptor> sources = new HashMap<>(env1.getTables());
```

Comment:
`final Map<String, TableDescriptor> tables`

Reply:
Good catch on the naming.